### PR TITLE
feat(i18n): add Turkish language file

### PR DESCRIPTION
**Pull request by iozcelik**

### CODE REVIEW

File: i18n/tr.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
+toggleMenu:
+    other: Menüyü Gizle
+
+darkMode:
+    other: Koyu Mod
+
+subtitle:
+    other: Biraz yazınca blog olacak
```
Comment (CaiJimmy) [inline i18n/tr.yaml:8]: That's not part of translation file.

**Review by CaiJimmy — CHANGES_REQUESTED** [review]
Thanks for your contribution. I found a translation string that shouldn't be here. Can you remove it please?

**Review by iozcelik — COMMENTED** [review]
Remove subtitle translate

File: i18n/tr.yaml (hunk 1 above)
Comment (iozcelik) [reply i18n/tr.yaml]: Yes, I removed it.

**Review by CaiJimmy — APPROVED** [review]

**Pull request merged**